Engine_Revit_UI/Create/Workset.cs

```diff
@@ -0,0 +1,88 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using Autodesk.Revit.DB;
+using BH.oM.Reflection.Attributes;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace BH.UI.Revit.Engine
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Creates a new workset with a specified name and sets the 'Workset' dropdown parameter to it.")]
+        [Input("document", "Revit model file")]
+        [Input("parameter", "Revit parameter")]
+        [Input("worksetName", "Name of the new workset")]
+        [Output("bool", "true for success, false for failure")]
```

Comment:
We are trying to be as neat and informative as possible when it comes to descriptions. This means:
- full sentences starting with capital and ending with a full stop
- reasonably rich information about arguments, e.g. "Name of the new workset to be added." instead of "Name of the new workset" etc.

Besides that, other remarks coming to my head:
- 'Workset' parameter is a dropdown only in Revit - on code level there is no dropdown 🙈 
- I would name the output "success" if it was a bool, but in case of returning `Workset` type I would probably  go for simple "workset"

Reply:
Revised accordingly